services/imageproxy/local.go

```diff
@@ -66,6 +67,20 @@ func makeLocalBackend(proxy *ImageProxy) *LocalBackend {
 	}
 }
 
+type contentTypeRecorder struct {
+	http.ResponseWriter
+	filename string
+}
+
+func (rec *contentTypeRecorder) WriteHeader(code int) {
+	hdr := rec.ResponseWriter.Header()
+	if hdr.Get("Content-Type") == "image/svg+xml" {
+		hdr.Set("Content-Disposition", "attachment;filename=\""+rec.filename+"\"")
```

Comment:
Do we know `rec.filename` won't contain double-quotes? If not, we should replace them with underscores.

Reply:
Used %q to do the quoting. Let me know if that doesn't work.